DocumentService/Controllers/DocumentsAnonymousController.cs

```diff
@@ -0,0 +1,151 @@
+﻿namespace DocumentService.Controllers
```

Comment:
The DocumentAnonymousController should be the controller without those auth; while DocumentController should be the controller with.

Reply:
Yi this is exactly what I did.
Not sure where do you see an authorization in the Anonymous controller ?

Please note that what you pointed out is the namespace used for all controllers. I think this is why you are referring to?

Reply:
I did not test, but refer with Asuquo's error, I quickly check the code, looks like two controllers are using same routes for endpoints, it might cause issue.  so we might need to change the routes for anonymouscontroller

Reply:
Weird, the Route is only the pre-route to access the controller. 
After that .Net has a convention name to access any controller by it's name. .Net will ignore the Controller part.
exp : DocumentAnonymous**Controller**. is accessed by api/DocumentAnonymous
For DocumentController, it shoudl be accessed by api/Document.

Reply:
As we define Route for controller, we need to add something like [Route("api/[controller]")] etc... into route to get controuller name into route. This is just my guess as I did not have solution, but you can do a quick testing locally to verify

Reply:
Good catch as always Yin. You are da man !
I did add the explicit route to the anonymous controller.
[Route("api/Anonymous")] 
Does this look good for the DocumentAnonymousController ?

Reply:
looks good to me. Have you tested it locally to verify the changes?

Reply:
All good now, tested with Auquo and working fine now.

Reply:
Yi, Could you do a final review please before the Merge?
It has been tested with Asuquo. Thanks